section4/reflection.md

```diff
@@ -1,22 +1,87 @@
+
 ## Section 4 Reflection
 
 1. How different did your workflow feel this week, considering we asked you to follow the Pomodoro technique?
+Time management is hard for me since I need to read not once, not twice but sometimes even three times the same information for me to be able to retain the information. I like the idea of having a timer to have a better flow with homework and learning in general. I just have to make sure I still learn what I need to to learn in that time sice I am learning while completing tasks.
```

Comment:
Turing can and will provide accommodations for students whose first language is not English and need extra time. I encourage you to advocate for that with your Mod 1 instructors early in the module if you'd like to have that!

Reply:
Thank you! I will make sure I do that.